.rspec

```diff
@@ -1,4 +1,3 @@
 --color
```

Comment:
Yeah I changed my mind on it, but since you agree I removed and gitignored it as well as moved the config to `spec_helper` as you suggested in 9490ad7.